src/style.css

```diff
@@ -0,0 +1,40 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.box .stars:nth-child(n) {
```

Comment:
you don't have `.box` in `html`, so this style will never work

Reply:
my bad